### DISCUSSION TITLE
Kong Rate limit plugin in HA mode

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8799] [Help]
Hi I am running kong helm 2.7 with two replicas of kong deployment.
And my underlying applications are also running with two replicas.
I want to add rate limit on the ingress with 10 requests per second config.
The the issue is rate limit is happening but on 40 th request, so it is taking consideration on 2 kong replicas and 2 application replicas.

**Note: My Kubernetes cluster is a multi node cluster**

Is there any way I can achieve on ingress level, irrespective of how many replicas I am running.

**Rate limit. yaml**

apiVersion: configuration.konghq.com/v1
kind: KongPlugin
metadata:
  name: rate-limit
config: 
  second: 10
  policy: local
  fault_tolerant: true
  hide_client_headers: false
plugin: rate-limiting

**Ingress.yaml**

apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: test-ingress-add-user
  annotations:
    kubernetes.io/ingress.class: kong
    konghq.com/plugins: rate-limit
  labels:
    app: test
spec:
  rules:
    - http:
        paths:
          - path: /test/addUser
            backend:
              serviceName: test-service
              servicePort: 8447
@hbagdi

**Comment by fffonion** [comment]
@guptadeepali39 You will need to use `cluster` or `redis` for `policy`. (https://docs.konghq.com/hub/kong-inc/rate-limiting/)
